### DISCUSSION TITLE
Slide3D Presentations & Editor using model-viewer

### DISCUSSION THREAD

**Discussion by meirlebel** [google/model-viewer#1806] [Show and tell]
Hi All,

Please visit my website [Slide3D](https://www.slide3d.com) which uses the model-viewer tech.
Thank you model-viewer team, waiting for more API.

Direct to presentations samples:
 [General](https://www.slide3d.com/presentation.php?name=demo-mv-site&id=f261bb05-212e-44d8-9e0d-6a13e3c209fb)
 [Mechanic](https://www.slide3d.com/presentation.php?name=mechaic-mv-site&id=994e0a10-260f-426d-9cce-83f9beeeb157)

My [editor](https://www.slide3d.com/editor.html)

Tn'x
Meir Lebel

**Comment by elalish** [comment]
Very cool, thanks for sharing! I love the shiny mechanical models; if you merge the normal vectors you'll turn the facets into nice smooth curves.

**Reply by meirlebel** [reply]
Tn'x elalish
